### PR TITLE
Move babel-preset-es2015 to devDependencies

### PR DESCRIPTION
**Pull request by morrisallison**
@yola/frontend-engineering

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: README.md [hunk 1]
```diff
@@ -65,8 +65,9 @@ Please view the [API reference][api] for details and more examples.
 
 A [Promises/A+][promise-spec] compliant promise implementation must be available globally.
 
-Consumers using [Browserify](browserify) must have the [Babelify](babelify) transform.
+Consuming packages using [Browserify][browserify] must have the [Babelify][babelify] transform, and the [ES2015 preset][babel-preset-es2015] available.
```
Comment (beck) [inline README.md]: Just use the copy ask your footnote key:

```
Consuming packages using [Browserify][] must have the [Babelify][] transform, and the [ES2015 preset][] available.

[Babelify]: https://github.com/babel/babelify
[Browserify]: http://browserify.org/
[ES2015 preset]: https://babeljs.io/docs/plugins/preset-es2015/
```

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by beck** [comment]
👍

**Pull request merged**